README.md

```diff
@@ -2,15 +2,6 @@
 
 A kde plasma accent color changer that changes some colors in current colorscheme.
 
-## Getting Started!
+Please Please accept my MR
 
-This project is a starting point for a Flutter application.
 
-A few resources to get you started if this is your first Flutter project:
-
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
-
```

Comment:
You deleted this line. In a real MR, devs would have questioned why you deleted this even if that is mentioned in the description.

Reply:
First get selected in gsoc then talk with me

Reply:
Okay